components/Tutorial/content/bustHoaxes.js

```diff
@@ -7,53 +9,53 @@ import tutorial6 from '../images/bust-hoaxes-tutorial-6.png';
 
 export default [
   {
-    title: '覺得查核太難了嗎？從覆核別人的查核回應開始吧！',
-    subTitle: '想寫出好的查核訊息前，先看看別人是怎麼查核的吧！',
-    content:
-      '「Cofacts 真的假的」上的訊息查核，是由世界各地的網友編輯，無償自主查核貢獻的喔！但是編輯們的查核成果，不見得就是正確完整的呢！因此，需要網友編輯們來覆核評價，協助篩選出好的查核結果。新手編輯們，也可以從中學習別人是怎麼查核闢謠呢！',
+    title: t`Do you feel fact-checking is too difficult? You can start by reviewing the responses of others.`,
+    subTitle: t`Take a look at how others do fact-checking may help you write good quality responses.`,
+    content: t`Reviews of information on Cofacts are collaborated by contributors from all over the world and free of charge. However, the fact-checking results may not necessarily be correct or complete all the time. That is why the project requires the role of editor to review the checking results to select responses of high quality. To novice editors, they may learn how others fact-check on rumors through the review process.`,
     subContent: [
       {
-        title: '尋找需要被覆核的回應',
+        title: t`Searching responses waiting to be reviewed`,
         content: [
           {
             type: 'text',
-            data: `「最新查核」會列出其他志工編輯查核回應，以下不同篩選能幫你篩選出——
-            「還未有效查核」：目前還沒有使用者覺得好的可疑訊息，推薦使用。
-            「熱門回報」：目前很多使用者想問真假的可疑訊息。
-            「熱門討論」：目前很多編輯查核回應的可疑訊息。`,
+            data: t`"Latest Checked" section lists all volunteer editors' reviews to fact-checking responses, you can filter information pieces with the tags as follows:
+            
+            "No validated responses yet": This labels the pieces of information that may lack validated high-quality responses. We recommend this tag as a start of searching.
+            
+            "Popularly Reported": This labels the suspicious messages that are trending among requests for fact-checking.
+            
+            "Hot Topics": This labels the suspicious messages that have been responded to and reviewed a lot.
```

Comment:
TODO (in another PR): We may need to update corresponding UI text accordingly.
![image](https://user-images.githubusercontent.com/108608/110197325-cd986980-7e85-11eb-9e84-ac4d332d2349.png)